config/custom-environment-variables.json

```diff
@@ -1,5 +1,6 @@
 {
   "launch-vehicle-fbm": {
+    "dashBotKey": "DASHBOT_KEY",
```

Comment:
after the refactor, it's the SDK/app's problem now though, not `conversationLogger`, so that's much easier to change in the future